release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -80,11 +80,18 @@ To view bug fixes, see the [GitHub milestone for 2201.x.0 (Swan Lake)](https://g
 
 #### Ballerina Update Tool
 
+#### GraphQL Tool
+- Added a new Ballerina script generation including the Ballerina data types related to the connector configuration
```

Comment:
IMO we can consider this as an improvement and say `Improved graphQL client config generation to support runtime configurability`

Reply:
Should be: `Improved the GraphQL client config generation to support runtime configurability`.